app/views/shared/_settings_nav.html.erb

```diff
@@ -7,6 +7,7 @@
     <li><%= link_to 'Feeds', settings_feeds_path, class: is_active?(['settings', 'subscriptions'], ['feeds', 'edit']) %></li>
     <li><%= link_to 'Actions', actions_path, class: is_active?('actions', ['index', 'new', 'edit']) %></li>
     <li><%= link_to 'Sharing', sharing_services_path, class: is_active?('sharing_services', 'index') %></li>
+    <li><%= link_to 'Applications', settings_applications_path, class: is_active?('applications', 'index') %></li>
```

Comment:
This could be updated to get an active class:

```diff
- <li><%= link_to 'Applications', settings_applications_path, class: is_active?('applications', 'index') %></li>
+ <li><%= link_to 'Applications', settings_applications_path, class: is_active?('settings', 'applications') %></li>
```